### ISSUE TITLE
Lint a whole role, not a single playbook

### ISSUE THREAD

**Issue by angristan** [ansible/ansible-lint-action#6]
I currently use this with Travis:

```yml
language: python
install:
  - pip install ansible-lint
  - ansible-lint --version
script:
  - ansible-lint .
```

But it seems this action wants a playbook file no matter what. Is there a way to lint a whole folder? In my case, it's a role.

**Comment by angristan** [comment]
Meanwhile I'm using this:

```yml
name: ansible-lint

on: [push]

jobs:
  build:
    runs-on: ubuntu-latest
    name: ansible-lint
    steps:
      - uses: actions/checkout@master
      - uses: actions/setup-python@v1
      - run: pip install ansible-lint
      - run: ansible-lint --version
      - run: ansible-lint .
```

**Comment by webknjaz** [comment]
FWIW we also need to add an `action.yml` file to support the new version of GitHub Actions CI/CD.

**Comment by webknjaz** [comment]
Resolved by #7.
